mathesar/rpc/tables.py

```diff
@@ -47,6 +47,26 @@ def list_(*, schema_oid: int, database_id: int, **kwargs) -> list[TableInfo]:
     ]
 
 
+@rpc_method(name="tables.get")
+@http_basic_auth_login_required
+@handle_rpc_exceptions
+def get_(*, table_oid: int, database_id: int, **kwargs) -> TableInfo:
```

Comment:
I don't understand why you went with `get_` here. `get` isn't a keyword, is it?

Reply:
You are right, I erroneously thought it was a keyword. I've changed it to `get`.